therapi-runtime-javadoc/src/main/java/com/github/therapi/runtimejavadoc/internal/parser/CommentParser.java

```diff
@@ -63,7 +63,7 @@ private static InlineLink createLinkElement(String value) {
         String classRef = ref[0];
         String memberRef = ref.length > 1 ? ref[1] : null;
 
-        Link link = new Link(label, classRef, memberRef);
+        Link link = new Link(label, classRef.isEmpty() ? null : classRef, memberRef);
```

Comment:
I wonder if there's a way to infer the class name from the enclosing elements?

Reply:
I'm now passing along the current class as fall-back. It's a little verbose passing it around to all the methods, but it works I guess.